packages/language-server/index.js

```diff
@@ -156,6 +147,20 @@ connection.onRequest('mdx/toggleStrong', async (parameters) => {
 
 connection.onInitialized(() => {
   server.initialized()
+  server.watchFiles([
+    `**/*.{${[
+      'cjs',
+      'cts',
+      'js',
+      'jsx',
+      'json',
+      'mdx',
+      'mjs',
+      'mts',
+      'ts',
+      'tsx'
```

Comment:
I think this is only needed when TypeScript is enabled, right? Otherwise just `mdx` should be sufficient.

Anyway, good enough for now. I can change this later if you agree.

Reply:
You're right! This should use `tsEnabled` as the condition.

Reply:
Resolved in https://github.com/mdx-js/mdx-analyzer/pull/432